nodes/pods/nodes-pods-autoscaling.adoc

```diff
@@ -66,5 +66,4 @@ include::modules/nodes-pods-autoscaling-status-viewing.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 == Additional resources
-* xref:../../applications/deployments/what-deployments-are.adoc#what-deployments-are[Understanding deployments and deployment configs].
-* https://cloud.redhat.com/blog/horizontal-pod-autoscaling-of-quarkus-application-based-on-memory-utilization[Horizontal Pod Autoscaling of Quarkus Application Based on Memory Utilization].
+* xref:../../applications/deployments/what-deployments-are.adoc#what-deployments-are[Understanding deployments and deployment configs]
```

Comment:
This lost the second link, is that purposeful? It looks to be missing the link prefix, does that just need to be added? Probably something to look out for elsewhere as well

```suggestion
* xref:../../applications/deployments/what-deployments-are.adoc#what-deployments-are[Understanding deployments and deployment configs]
* link:https://cloud.redhat.com/blog/horizontal-pod-autoscaling-of-quarkus-application-based-on-memory-utilization[Horizontal Pod Autoscaling of Quarkus Application Based on Memory Utilization]
```